plugin.json

```diff
@@ -0,0 +1,22 @@
+  
+{
+    "id": "tesseract-ocr",
+
+    "version": "1.0.0",
+
+    "meta": {
+        "label": "Tesseract ocr",
+
+        "description": "Plugin for optical character recognition (OCR) in python using the tesseract engine.",
+
+        "author": "Dataiku (Stanislas GUINEL)",
+
+        "icon": "icon-folder-close",
+
+        "tags": ["NLP", "Images"],
+
+        "url": "https://www.dataiku.com/product/plugins/tesseract-ocr.html",
```

Comment:
The url page is not well formatted, use https://www.dataiku.com/product/plugins/tesseract-ocr/ instead